product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -34,7 +34,7 @@ Prior to creating your cluster, make sure you have enough resources. Without eno
 
      - [Single node](/biganimal/latest/overview/02_high_availability/#single-node) creates a cluster with one primary and no standby replicas. Suited for test environments where high availability might not be required. You can create single node clusters running EDB Postgres Advanced Server or PostgreSQL. 
 
-    -  [High availability](/biganimal/latest/overview/02_high_availability/#high-availability) creates a cluster with one primary and two standby replicas in different availability zones. You can create high availability clusters running EDB Postgres Advanced Server or PostgreSQL. 
+    -  [High availability](/biganimal/latest/overview/02_high_availability/#high-availability) creates a cluster with one primary and two standby replicas in different availability zones. You can create high availability clusters running EDB Postgres Advanced Server or PostgreSQL. Only High availability clusters allow you to enable read-only connections for users. However, enabling read-only connections may raise infrastructure costs by about $.025 an hour due to load balancing. 
```

Comment:
```suggestion
    -  [High availability](/biganimal/latest/overview/02_high_availability/#high-availability) creates a cluster with one primary and multiple replicas in different availability zones. You can create High Availability clusters running EDB Postgres Advanced Server or PostgreSQL. Only High Availability clusters allow you to enable read-only connections. However, enabling read-only connections will increase infrastructure costs by about $.025 an hour for the additional load balancing rule, plus traffic charges. 
```

Reply:
Thanks for your comments, Aaron. We are making some slight modifications to align with our style guide. For example, we minimize the use of capitalization and try to avoid future tense. Let me know if you have any concerns with our modifications.

Reply:
No problem at all, simply suggestions!  Perhaps then "Only High" needs to be "Only high" in the original.

Reply:
You are correct and Chris is planning to make that change.